src/scss/grommet-core/_objects.search-input.scss

```diff
@@ -1,53 +1,54 @@
 // (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP
 
+.grommet .search-input,
```

Comment:
Added a comment explaining reasoning for .grommet prefix to .search-input__input selector.